simple_repository_browser/model.py

```diff
@@ -146,12 +146,12 @@ async def project_query(
                 )
 
             # Main query uses the builder's complete query method
-            query, params = sql_builder.build_complete_query(
+            sql_query, params = sql_builder.build_complete_query(
                 "SELECT canonical_name, summary, release_version, release_date FROM projects",
                 page_size,
                 offset,
             )
-            results = cursor.execute(query, params).fetchall()
+            results = cursor.execute(sql_query, params).fetchall()
```

Comment:
This fixed a bug which was caused by this variable overriding the original input, and the SQL query showing up in the search results.